src/lib/components/CopyCode/index.js

```diff
@@ -21,8 +21,18 @@ class CopyCode extends BaseElement {
 
     if (!this.copyButton) {
       this.copyButton = document.createElement("button");
-      this.copyButton.className = "web-copy-code__button";
+      this.copyButton.className =
+        "w-button--icon w-button--round web-copy-code__button";
+      this.copyButton.setAttribute("data-icon", "file_copy");
+      this.copyButton.setAttribute("aria-label", "Copy code");
```

Comment:
It took some reading to understand that the `span` tooltip only shows on hover. Maybe a quick comment: we set `aria-label` and not `title` because `title` would result in both the browser's tooltip and ours showing.

Reply:
Actually, I'm setting `aria-label` because `title` isn't accessible to sighted keyboard users. I'll note that.